installing/installing_azure/upi/installing-azure-preparing-upi.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-azure-preparing-upi"]
+= Preparing to install a cluster on Azure
+include::_attributes/common-attributes.adoc[]
+:context: installing-azure-preparing-upi
+
+toc::[]
+
+To prepare for installation of an {product-title} cluster on {azure-short}, complete the following steps:
+
+* You have xref:../../../installing/installing-preparing.adoc#installing-preparing[selected a cluster installation method].
```

Comment:
```suggestion
* You xref:../../../installing/installing-preparing.adoc#installing-preparing[selected a cluster installation method].
```

Reply:
As per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites), this is a valid usage.